### ISSUE TITLE
Event paper-header-transform not fired on header changes caused by window resizes

### ISSUE THREAD

**Issue by ramonwirsch** [PolymerElements/paper-scroll-header-panel#48]
If the window is resized, the length of the main content can change, and therefore the height of the header.
We use the 'paper-header-transform' event to scale a picture and title inside the header, but this event is not fired on resizes, so the header contents may misalign until the next scroll event.

I have found no easy way to manually force it to refresh.

**Comment by blasten** [comment]
@ramonwirsch `iron-resize` triggers when you resize the window and it should update the height of the header. The method `measureHeaderHeight` is called on every resize event. https://github.com/PolymerElements/paper-scroll-header-panel/blob/master/paper-scroll-header-panel.html#L383, but potentially the header had a fixed height. Can you provide a jsbin to see if there's anything this element could handle?

**Comment by ramonwirsch** [comment]
In the meantime I managed to find the seperate resize event on my own and fixed my problem.
I also just tried to recreate the circumstances of when i opened the issue and for life of me I cannot figure out what went wrong that day, as it is working perfectly right now
I apologize for the inconvenience. :-(

But just as a suggestion: given the event name "header-transform", the event could additionally be fired on changes in width of the header. It would make using the available width to switch to shorter headlines and smaller logos much easier...
